covalent/_results_manager/result.py

```diff
@@ -303,37 +299,22 @@ def get_node_result(self, db: DataStore, node_id: int) -> dict:
                             - stderr: The stderr of the node execution.
         """
 
-        # return {
-        #     "node_id": node_id,
-        #     "node_name": self.lattice.transport_graph.get_node_value(node_id, "node_name"),
-        #     "start_time": self.lattice.transport_graph.get_node_value(node_id, "start_time"),
-        #     "end_time": self.lattice.transport_graph.get_node_value(node_id, "end_time"),
-        #     "status": self.lattice.transport_graph.get_node_value(node_id, "status"),
-        #     "output": self.lattice.transport_graph.get_node_value(node_id, "output"),
-        #     "error": self.lattice.transport_graph.get_node_value(node_id, "error"),
-        #     "sublattice_result": self.lattice.transport_graph.get_node_value(
-        #         node_id, "sublattice_result"
-        #     ),
-        #     "stdout": self.lattice.transport_graph.get_node_value(node_id, "stdout"),
-        #     "stderr": self.lattice.transport_graph.get_node_value(node_id, "stderr"),
-        # }
-
         return {
             "node_id": node_id,
-            "node_name": self._get_node_name(db, node_id),
+            "node_name": self.lattice.transport_graph.get_node_value(node_id, "node_name"),
```

Comment:
Will there be time when there can be a lookup failure i.e "node_name" not present, to cover that can we return None/something equivalent?
```suggestion
            "node_name": self.lattice.transport_graph.get_node_value(node_id, "node_name") or None
```